src/relay/pass/type_infer.cc

```diff
@@ -61,6 +61,19 @@ TVM_REGISTER_API("tvm.relay.type_relation.TupleGetItem")
 .set_body_typed<bool(const Array<Type>&, int, const Attrs&, const TypeReporter&)>(
     TupleGetItemRel);
 
+struct ResolvedTypeInfo {
+  explicit ResolvedTypeInfo(Type checked_type, Array<Type> type_args)
+      : checked_type(checked_type), type_args(type_args) {}
+  ResolvedTypeInfo(const ResolvedTypeInfo& rti)
+      : checked_type(rti.checked_type), type_args(rti.type_args) {}
+  ResolvedTypeInfo() : checked_type() {}
```

Comment:
no need to initalized checked_type, it will pick the default constructor

Reply:
I added these constructors due to a template instantiation error earlier.